src/PureCloud.Client/Apis/TextbotsApi.cs

```diff
@@ -41,9 +40,9 @@ public async Task<BotSearchResponseEntityListing> GetTextbotsBotsSearchAsync(IEn
 
         if (botId != null)
         {
-            foreach (var item in botId)
+            foreach (var botIdItem in botId)
```

Comment:
The botId variable should be botIds and botIdItem should be botId.

Anytime there is a variable that accepts a collection of variable, make it ienumerabe then it should be plural form. When looping over the collection, each iteration should be named the single form of the collection. Please apply this everywhere